nixos/modules/virtualisation/containers.nix

```diff
@@ -367,6 +370,16 @@ in
               '';
             };
 
+            additionalCapabilities = mkOption {
+              type = types.listOf types.str;
+              default = [];
+              example = [ "CAP_NET_ADMIN" "CAP_MKNOD" ];
+              description = ''
+                Grant additional capabilities to the container.  See the
+                capabilities(7) man page for a list of capabilities.
```

Comment:
I would also mention `systemd-nspawn(1)`, because it explains, which capabilities are granted by default.